sdk/src/main/java/com/horizen/transaction/mainchain/SidechainCreation.java

```diff
@@ -63,8 +58,8 @@ public ForgerBox getBox() {
         );
     }
 
-    public byte[] getBackwardTransferPoseidonRootHash() {
-        return new byte[0];
+    public byte[] genSysConstant() {
```

Comment:
Probably,  `genSysConstant` -> `getSysConstant`

Reply:
we use the same name `genSysConstant` everywhere, so it's not missprint

Reply:
ok, i got it. Thought that `gen` is `generate`, but it's actually like get generated sys constant